Description: ftbfs work around WebCore::QtPlatformPlugin has no member named
             createFullScreenVideoHandler
Author: Fathi Boudra <fabo@debian.org>
--- a/WebCore/WebCore.pro
+++ b/WebCore/WebCore.pro
@@ -2572,13 +2572,11 @@ contains(DEFINES, ENABLE_VIDEO=1) {
         bindings/js/JSAudioConstructor.cpp
 
         !lessThan(QT_MINOR_VERSION, 6):contains(MOBILITY_CONFIG, multimedia) {
-            HEADERS += \ 
-                platform/graphics/qt/MediaPlayerPrivateQt.h \
-                $$PWD/../WebKit/qt/WebCoreSupport/FullScreenVideoQt.h
+            HEADERS += \
+                platform/graphics/qt/MediaPlayerPrivateQt.h
 
             SOURCES += \
-                platform/graphics/qt/MediaPlayerPrivateQt.cpp \
-                $$PWD/../WebKit/qt/WebCoreSupport/FullScreenVideoQt.cpp
+                platform/graphics/qt/MediaPlayerPrivateQt.cpp
 
             CONFIG *= mobility
             MOBILITY += multimedia
